src/Uno.UI.Runtime.Skia.Wpf/Themes/Generic.xaml

```diff
@@ -53,4 +54,40 @@
 			</Setter.Value>
 		</Setter>
 	</Style>
+
+	<!--<Style TargetType="{x:Type wpf:PasswordBox}">
+		<Setter Property="SnapsToDevicePixels" Value="True" />
+		<Setter Property="KeyboardNavigation.TabNavigation" Value="None" />
+		<Setter Property="FocusVisualStyle" Value="{x:Null}" />
+		<Setter Property="MinWidth" Value="0" />
+		<Setter Property="MinHeight" Value="0" />
+		<Setter Property="AllowDrop" Value="true" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="{x:Type wpf:PasswordBox}">
+					<Border Name="Border"
+							CornerRadius="0"
+							Margin="{TemplateBinding Margin}"
+							Padding="{TemplateBinding Padding}"
+							BorderThickness="0">
+						<Border.Background>
+							<SolidColorBrush Color="Transparent" />
+						</Border.Background>
+						<Border.BorderBrush>
+							<SolidColorBrush Color="Transparent" />
+						</Border.BorderBrush>
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="CommonStates">
+								<VisualState x:Name="Normal" />
+								<VisualState x:Name="Disabled" />
+								<VisualState x:Name="ReadOnly" />
+								<VisualState x:Name="MouseOver" />
+							</VisualStateGroup>
+						</VisualStateManager.VisualStateGroups>
+						<ScrollViewer Margin="0" x:Name="PART_ContentHost" />
+					</Border>
+				</ControlTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>-->
```

Comment:
This is a copy from `WpfTextViewTextBox` styling. I thought this would fix the styling issue, but it doesn't. (Commenting/Uncommenting this code has no effect)

Reply:
@MartinZikmund It looks like `WpfTextViewTextBox.Style` is null. So, the styles already match (both are null)